core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/clazz/edit/ClazzEditViewModel.kt

```diff
@@ -392,17 +427,19 @@ class ClazzEditViewModel(
 
     fun onCheckedAttendanceChanged(checked: Boolean) {
         onEntityChanged(_uiState.value.entity?.shallowCopy {
-            clazzFeatures = if(checked) {
+            clazzFeatures = if (checked) {
```

Comment:
As mentioned - please don't change the spacing when nothing else has changed - just makes code review harder.